standalone-kafka/docker-compose.yml

```diff
@@ -1,44 +1,47 @@
 ---
-version: '3'
-
 services:
-  zookeeper:
-    image: ${DOCKER_REGISTRY:-docker.io}/confluentinc/cp-zookeeper:${ZOOKEEPER_VERSION:-latest}
-    container_name: zookeeper
-    hostname: zookeeper
-    environment:
-      TZ: ${TIMEZONE:-America/New_York}
-      ZOOKEEPER_CLIENT_PORT: 2181
-      ZOOKEEPER_TICK_TIME: 2000
-
   broker:
-    image: ${DOCKER_REGISTRY:-docker.io}/confluentinc/cp-kafka:${KAFKA_VERSION:-latest}
-    container_name: broker
+    container_name: kafka
+    image: quay.io/strimzi/kafka:latest-kafka-3.7.0
```

Comment:
Any reason to change to `quay.io` instead of `${DOCKER_REGISTRY}` ?

Could perhaps make it a commented-out option.

Reply:
Just avoiding the DockerHub pull request limits.

Reply:
I think it makes sense to make it consistent as with the rest of the image references. I'll fix that and let you know. 👍